src/third-party/PhishingController.ts

```diff
@@ -194,11 +200,13 @@ export class PhishingController extends BaseController<
       return;
     }
 
+    this.lastFetched = Date.now();
```

Comment:
Hmmm. This is an improvement in that it should prevent duplicate config update requests. But it would now consider the config as having been updated even if it wasn't, and the request failed.

I guess this is fine for now. We already didn't handle that well.